internal/super/project.go

```diff
@@ -112,6 +113,19 @@ func (p *project) deploy() {
 	printDeployment(deployed, err, p.pathNameLookup)
 }
 
+func (p *project) AddSuperGeneratedAccount(name string) {
```

Comment:
This function is a bit light, I would avoid it and just use append directly where you need to.